addons/web/static/src/search/control_panel/control_panel.scss

```diff
@@ -28,6 +28,10 @@
             min-width: MIN(600px, 33%);
         }
     }
+
+    .o_cp_switch_buttons .btn {
+        z-index: 0;
+    }
```

Comment:
I would add a comment 
and maybe also force z-index: 0 when the btn is focused

Reply:
@aab-odoo I don't think it's possible to focus the button, I tried on master and I can't do anything with it

Reply:
I did it yesterday with the cross in the facet